client/js/components/GameView/Card.vue

```diff
@@ -194,4 +194,15 @@ export default {
     opacity: 0;
   }
 }
+
+@media (max-width: 600px) {
+  .player-card {
+    max-height: 10vh;
+    max-width: calc(10vh / 1.45);
+    &.glasses {
+      max-width: 10vh;
+      height: calc(10vh / 1.45);
+    }
+  }
```

Comment:
I would like to move cards to a width based implementation, using height feels weird though there may have been a reason for it. @itsalaidbacklife any idea?

Reply:
Yeah I'm open to that. I think the general idea was to ensure that the grid took up the full height of the viewport, but there's no good reason why that really needs to apply to the cards specifically

Reply:
Another issue I can write up this week! Good call